src/docs/sphinx/quickstart.rst

```diff
@@ -116,17 +116,17 @@ Serac.
 
 .. code-block:: bash
 
-   $ python3 scripts/uberenv/uberenv.py --prefix=../serac_libs
+   $ python3 scripts/llnl/build_tpls.py --prefix=../serac_libs
```

Comment:
@white238 do you think we should say `build_tpls.py` in quickstart as opposed to running simply `uberenv.py`?

Reply:
(note: i changed prefix to directory in the next commit)

Reply:
`build_tpls.py` is for building all the specs on the given platform. That sometimes doesn't make sense.

Reply:
ill change it back then